services/src/northbound/src/main/java/org/openkilda/northbound/service/impl/FlowServiceImpl.java

```diff
@@ -268,15 +257,16 @@ public FlowPayload updateFlow(final FlowPayload flow) {
     public List<FlowPayload> getFlows() {
         final String correlationId = RequestCorrelationId.getId();
         LOGGER.debug("Get flows request processing");
-        FlowGetRequest data = new FlowGetRequest();
+        BidirectionalFlowRequest data = new BidirectionalFlowRequest();
```

Comment:
There is no more "unidirectional request", so we can call it FlowReadRequest (and response - FlowReadResponse).